stdlib/regex.gr

```diff
@@ -678,68 +815,115 @@ parseRangeRest = (buf: RegExBuf, rng, spanFrom: Option<Number>, mustSpanFrom: Op
         if (!moreN(buf, 1)) {
           Err(parseErr(buf, "Missing closing `]`", 1))
         } else {
-          match(peekN(buf, 1)) {
+          match (peekN(buf, 1)) {
             Err(e) => Err(e),
             Ok(']') => {
-              match(mustSpanFrom) {
-                Some(_) => Err(parseErr(buf, "misplaced hyphen within square brackets in pattern", 1)),
+              match (mustSpanFrom) {
+                Some(_) =>
+                  Err(
+                    parseErr(
+                      buf,
+                      "misplaced hyphen within square brackets in pattern",
+                      1,
+                    ),
+                  ),
                 None => {
                   ignore(eat(buf, '-'))
                   ignore(eat(buf, ']'))
-                  match(rangeAddCaseAware(rng, spanFrom, buf.config)) {
+                  match (rangeAddCaseAware(rng, spanFrom, buf.config)) {
                     Err(e) => Err(e),
-                    Ok(rng) => Ok(rangeAdd(rng, Char.code('-')))
+                    Ok(rng) => Ok(rangeAdd(rng, Char.code('-'))),
                   }
-                }
+                },
               }
             },
-            Ok(_) when Option.isNone(spanFrom) => Err(parseErr(buf, "misplaced hyphen within square brackets in pattern", 1)),
+            Ok(_) when Option.isNone(spanFrom) =>
+              Err(
+                parseErr(
+                  buf,
+                  "misplaced hyphen within square brackets in pattern",
+                  1,
+                ),
+              ),
             Ok(_) => {
               ignore(eat(buf, '-'))
               parseRangeRest(buf, rng, None, spanFrom)
-            }
+            },
           }
         }
       },
       Ok('\\') => {
         ignore(eat(buf, '\\'))
-        if (!(buf.config.isPerlRegExp)) {
+        if (!buf.config.isPerlRegExp) {
           parseRangeRestSpan(buf, Char.code('\\'), rng, spanFrom, mustSpanFrom)
         } else {
           if (!more(buf)) {
-            Err(parseErr(buf, "escaping backslash at end pattern (within square brackets)", 0))
+            Err(
+              parseErr(
+                buf,
+                "escaping backslash at end pattern (within square brackets)",
+                0,
+              ),
+            )
           } else {
-            match(peek(buf)) {
+            match (peek(buf)) {
               Err(e) => Err(e),
-              Ok(c) when ((Char.code('a') <= Char.code(c) && Char.code(c) <= Char.code('z')) || (Char.code('A') <= Char.code(c) && Char.code(c) <= Char.code('Z'))) => {
-                match(mustSpanFrom) {
-                  Some(_) => Err(parseErr(buf, "misplaced hyphen within square brackets in pattern", 0)),
+              Ok(c) when Char.code('a') <= Char.code(c) &&
+              Char.code(c) <= Char.code('z') ||
```

Comment:
These should be indented since they're a part of the `when`

Reply:
Done